common-compose.yml

```diff
@@ -1,35 +1,39 @@
 # MySQL designed for container-native deployment on Joyent's Triton platform.
 
-mysql:
-    image: autopilotpattern/mysql:latest
-    mem_limit: 4g
-    restart: always
-    env_file: _env
-    environment:
-      - CONTAINERBUDDY=file:///etc/containerbuddy.json
-      - MANTA_PRIVATE_KEY
+version: '2'
 
-    # expose for linking, but each container gets a private IP for
-    # internal use as well
-    expose:
-      - 3306
+services:
 
-# ---------------------------------------------------
-# Consul as a service discovery tier
+    mysql:
+        image: autopilotpattern/mysql:latest
+        mem_limit: 4g
+        restart: always
+        env_file: _env
```

Comment:
Worth noting here that `_env` becomes relative to the repo root, so all subdirectories share a single `_env` file, which may or may not be a good idea long term (since I found in my own testing that having my local Swarm try to use the same Manta bucket as a cluster running on Triton caused headaches due to restoring the Triton cluster's and thus fighting over the bucket).